recipes-debian/vsftpd/vsftpd_debian.bb

```diff
@@ -0,0 +1,135 @@
+# base recipe: meta-networking/recipes-daemons/vsftpd/vsftpd_3.0.3.bb
+# base branch: warrior
+# base commit: a24acf94d48d635eca668ea34598c6e5c857e3f8
+SUMMARY = "Very Secure FTP server"
+HOMEPAGE = "https://security.appspot.com/vsftpd.html"
+SECTION = "net"
+LICENSE = "GPL-2.0-with-OpenSSL-exception"
+LIC_FILES_CHKSUM = "file://COPYING;md5=a6067ad950b28336613aed9dd47b1271 \
+                        file://COPYRIGHT;md5=04251b2eb0f298dae376d92454f6f72e \
+                        file://LICENSE;md5=654df2042d44b8cac8a5654fc5be63eb"
+
+inherit debian-package
+require recipes-debian/sources/vsftpd.inc
+
+DEPENDS = "libcap openssl"
+
+SRC_URI += "file://makefile-destdir.patch \
+            file://makefile-libs-and-strip.patch \
+            file://change-secure_chroot_dir.patch \
+            file://volatiles.99_vsftpd \
+            file://0001-vsftpd-allow-syscalls-in-the-seccomp-sandbox.patch \
+            ${@bb.utils.contains('DISTRO_FEATURES', 'pam', '', '${NOPAM_SRC}', d)} \
+            file://0001-sysdeputil.c-Fix-with-musl-which-does-not-have-utmpx.patch \
+            "
+
+PACKAGECONFIG ??= "tcp-wrappers"
+PACKAGECONFIG[tcp-wrappers] = ",,tcp-wrappers"
+
+DEPENDS += "${@bb.utils.contains('DISTRO_FEATURES', 'pam', 'libpam', '', d)}"
+RDEPENDS_${PN} += "${@bb.utils.contains('DISTRO_FEATURES', 'pam', 'pam-plugin-listfile', '', d)}"
+RDEPENDS_${PN} += "bash"
+PAMLIB = "${@bb.utils.contains('DISTRO_FEATURES', 'pam', '-L${STAGING_BASELIBDIR} -lpam', '', d)}"
+WRAPLIB = "${@bb.utils.contains('PACKAGECONFIG', 'tcp-wrappers', '-lwrap', '', d)}"
+NOPAM_SRC ="${@bb.utils.contains('PACKAGECONFIG', 'tcp-wrappers', 'file://nopam-with-tcp_wrappers.patch', 'file://nopam.patch', d)}"
```

Comment:
Applying nopam-with-tcp_wrappers.patch patch failed.

```
ERROR: vsftpd-3.0.3-r0 do_patch: Command Error: 'quilt --quiltrc /home/masami/emlinux/build/tmp-glibc/work/aarch64-emlinux-linux/vsftpd/3.0.3-r0/recipe-sysroot-native/etc/quiltrc push' exited with 0  Output:
Applying patch nopam-with-tcp_wrappers.patch
patching file builddefs.h
Hunk #1 FAILED at 2.
1 out of 1 hunk FAILED -- rejects in file builddefs.h
Patch nopam-with-tcp_wrappers.patch does not apply (enforce with -f)
ERROR: vsftpd-3.0.3-r0 do_patch: 
ERROR: vsftpd-3.0.3-r0 do_patch: Function failed: patch_do_patch
ERROR: Logfile of failure stored in: /home/masami/emlinux/build/tmp-glibc/work/aarch64-emlinux-linux/vsftpd/3.0.3-r0/temp/log.do_patch.9736
ERROR: Task (/home/masami/emlinux/build/../repos/meta-debian-extended/recipes-debian/vsftpd/vsftpd_debian.bb:do_patch) failed with exit code '1'
NOTE: Tasks Summary: Attempted 1972 tasks of which 1966 didn't need to be rerun and 1 failed.
```

Reply:
Since VSF_BUILD_TCPWRAPPERS and VSF_BUILD_SSL are defined in the debian patch, fixed the nopam-with-tcp_wrappers.patch and recipe accordingly.